diff --git a/configure b/configure
index bb8fedd..61673ba 100755
--- a/configure
+++ b/configure
@@ -2208,7 +2208,7 @@ $as_echo "$as_me: -----------------------------------------------" >&6;}
 . ./configure.user
 
 # check command line arguments and override the values set in configure.user with them
-for arg in $ac_configure_args
+for arg in "$(echo "$ac_configure_args" | tr "' '" "'\n'")"
 do
   stripped_arg=$(echo $arg | sed "s/^\('\)\(.*\)\1\$/\2/g")
   case $stripped_arg in
